way_to_home/static/src/components/map/map.css

```diff
@@ -0,0 +1,14 @@
+.map{
+    width: 100%;
+    height: 100%;
+    zIndex: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    margin: 0;
+    padding: 0;
```

Comment:
for properties margin and padding, zero is value by default

Reply:
ok